### DISCUSSION TITLE
Looking for Data from a Sensor (Newbie at work)

### DISCUSSION THREAD

**Discussion by HotteX** [mak-gitdev/HA_enoceanmqtt#32] [Q&A] [answered]
First thing: This Project made me come back to HA, because I gave up hope after years of waiting for a real support of enocean and turned to the Java-Side. So... thank you :)

I'm still struggling a bit with the start so I hope its a simple "Problem": 

I start my setup with 3 types of temperature sensors, 1 Nodon, 2 Eltako (Config below) running current Dev-Version on HAOS on a Pi with an EnoceanHAT. 

Eltako_2 worked out of the box. 
Eltako_1 does not (no entities), which is expected I assume, as it is not yet supported (I wrote a support request).

Now my real question to Nodon:
It should be supported, the entities are there but without values and no telegrams show up in the log just as it would never send one. 
With the HA-internal integration of Enocean this sensor works fine, also showing up in the logs. 
Unfortunately I don't have a second system to catch telegrams, but is there a way to see "raw" data to troubleshoot this issue?

```
[Temp/Eltako_1]
address         = 0xFFB17E00
rorg            = 0xA5
func            = 0x10
type            = 0x06

[Temp/Eltako_2]
address         = 0x0593DA67
rorg            = 0xA5
func            = 0x04
type            = 0x02

[Temp/Nodon]
address         = 0x050C4C89
rorg            = 0xA5
func            = 0x04
type            = 0x01
```

**Comment by mak-gitdev** [comment, marked as answer]
Hi @HotteX,

Happy to make you comeback to HA :+1: 

Regarding your Nodon sensor, could you please run the addon with `log_packets` and `debug` enabled. Then press the learn button on your sensor and check in the log file `enoceanmqtt.log` that there are messages from address `0x050C4C89` ?

If no message after pressing the learn button, then it would mean that your sensor has not enough power to send telegrams.

**Reply by HotteX** [reply]
Thank you for the info that it really has to be inside that log. I retried again and I found the error:

The solar panel solderpoints got loose, so what happened: 
It worked with HA_enocean.
I switched to enoceanmqtt, moved the sensor to press the button and the panel shifted a mm. It did not work. 
I switched back to HA_E, moved it and the panel shifted back: It works again.
Now I doublechecked and I saw the panel shifting. 

Case closed. Sorry :)
